test/settings.ts

```diff
@@ -83,7 +85,8 @@ const lotto = {
       'Cannot be created if the current lotto are not finished.',
     config_new_lottery_when_previous_lottery_not_finished:
       'Cannot be config if the current lotto are not finished.',
-    invalid_random_caller: 'Only Lottery can call function'
+    invalid_random_caller: 'Only Lottery can call function',
+    do_not_have_unclaimed_reward: "You don't have unclaimed rewards."
```

Comment:
This message is a bit opposite meaning. I am not sure, it should be "You don't have any rewards for claim"